parsl/executors/high_throughput/executor.py

```diff
@@ -152,6 +157,7 @@ def __init__(self,
                  working_dir: Optional[str] = None,
                  worker_debug: bool = False,
                  cores_per_worker: float = 1.0,
+                 mem_per_worker: float = 1.0,
```

Comment:
default to 1.0 or None? None seems better as the docstring mentions.

Reply:
I've set this to Optional[float] = None